lib/library.rb

```diff
@@ -0,0 +1,52 @@
+class Library
+  attr_reader :name,
+              :books,
+              :authors,
+              :checked_out_books,
+              :popularity
+  def initialize(name)
+    @name = name
+    @books = []
```

Comment:
I'd rather this be a method that's calculated dynamically based off of @authors. We want a single source of truth.